<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_PopupDial_NotesInfos</class>
 <widget class="QWidget" name="C_PopupDial_NotesInfos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>154</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label_Property">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Propriétés des notes personnelles</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="Line" name="line">
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>3</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_Title">
     <property name="text">
      <string>Titre de la note </string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_Titre"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_Author">
     <property name="text">
      <string>Ajouté par</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_Author"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="label_DateTime">
     <property name="text">
      <string>Date de rédaction</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QDateTimeEdit" name="dateTimeEdit_note"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
